### DISCUSSION TITLE
Issues in building sqitch using the docker file:  https://github.com/sqitchers/docker-sqitch/blob/main/Dockerfile

### DISCUSSION THREAD

**Discussion by PrakashSelvarajS** [theory/sqitch#746] [Q&A]
Hi, 
We have been using the Sqitch docker file in the github repo: https://github.com/sqitchers/docker-sqitch/blob/main/Dockerfile

It had been working well, but in the last 2 days we get a Sqitch docker build failure with error as below:

Can you please help resolve this error?

Error:
====

#16 23.39 #   Failed test 'Destination should omit password'
#16 23.39 #   at t/snowflake.t line 321.
#16 23.39 #          got: 'db:snowflake://fred@foo.snowflakecomputing.com/try?warehouse=foo;role=yup'
#16 23.39 #     expected: 'db:snowflake://fred:@foo.snowflakecomputing.com/try?warehouse=foo;role=yup'
#16 23.41 # Looks like you failed 1 test of 128.

Detailed logs with error seen as below:
==========================

#16 [sqitch-build 6/6] RUN perl Build.PL --quiet --install_base /app --etcdir /etc/sqitch     --config installman1dir= --config installsiteman1dir= --config installman3dir= --config installsiteman3dir=     --with sqlite --with postgres --with firebird --with odbc     && ln -s  /usr/include/ibase.h /usr/include/firebird/     && ./Build test && ./Build bundle     && rm -rf /app/man     && find /app -name '*.pod' | grep -v sqitch | xargs rm -rf
#16 1.382 t/add.t ............. ok
#16 1.958 t/base.t ............ ok
#16 2.345 t/blank.t ........... ok
#16 3.110 t/bundle.t .......... ok
#16 3.667 t/change.t .......... ok
#16 4.262 t/changelist.t ...... ok
#16 4.792 t/check.t ........... ok
#16 5.384 t/checkout.t ........ ok
#16 6.098 t/command.t ......... ok
#16 6.948 t/config.t .......... ok
#16 7.188 t/configuration.t ... ok
#16 7.505 t/conn_cmd_role.t ... ok
#16 7.829 t/cx_cmd_role.t ..... ok
#16 8.187 t/datetime.t ........ ok
#16 8.685 t/depend.t .......... ok
#16 9.261 t/deploy.t .......... ok
#16 9.993 t/engine.t .......... ok
#16 11.27 t/engine_cmd.t ...... ok
#16 11.91 t/exasol.t .......... ok
#16 12.46 t/firebird.t ........ ok
#16 12.81 t/help.t ............ ok
#16 13.54 t/init.t ............ ok
#16 14.09 t/item_formatter.t .. ok
#16 14.59 t/linelist.t ........ ok
#16 15.27 t/log.t ............. ok
#16 15.93 t/mooseless.t ....... ok
#16 16.59 t/mysql.t ........... ok
#16 17.00 t/options.t ......... ok
#16 17.61 t/oracle.t .......... ok
#16 18.20 t/pg.t .............. ok
#16 19.16 t/plan.t ............ ok
#16 19.77 t/plan_cmd.t ........ ok
#16 20.10 t/pragma.t .......... ok
#16 20.75 t/rebase.t .......... ok
#16 21.37 t/revert.t .......... ok
#16 22.25 t/rework.t .......... ok
#16 22.80 t/show.t ............ ok
#16 23.39 
#16 23.39 #   Failed test 'Destination should omit password'
#16 23.39 #   at t/snowflake.t line 321.
#16 23.39 #          got: 'db:snowflake://fred@foo.snowflakecomputing.com/try?warehouse=foo;role=yup'
#16 23.39 #     expected: 'db:snowflake://fred:@foo.snowflakecomputing.com/try?warehouse=foo;role=yup'
#16 23.41 # Looks like you failed 1 test of 128.
#16 23.43 t/snowflake.t ....... 
#16 23.43 Dubious, test returned 1 (wstat 256, 0x100)
#16 23.43 Failed 1/128 subtests 

Thank you

Regards,
Prakash Selvaraj
(swathyprakash3012@gmail.com)

**Comment by theory** [comment]
See sqitchers/sqitch#744 for the underlying cause. Probably should just remove `./Build test` from the Dockerfile.

**Comment by PrakashSelvarajS** [comment]
Thank you a lot. This helped us.
